src/Utilities/CssConcatenator/RuleBlock.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Utilities\CssConcatenator;
+
+/**
+ * Implementation of media rules sub-property for \Pelago\Emogrifier\Utilities\CssConcatenator
+ *
+ * @internal
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ * @author SignpostMarv
+ */
+class RuleBlock
+{
+    /**
+     * Array whose keys are selectors for the rule block (values are of no
+     *     significance);
+     *
+     * @var mixed[]
+     *
+     * @psalm-var array<string, mixed>
+     */
+    public $selectorsAsKeys = [];
+
+    /**
+     * The property declarations, e.g. "margin-top: 0.5em; padding: 0".
+     *
+     * @var string
+     */
+    public $declarationsBlock;
```

Comment:
possible `@todo` docblock tags:
* amend to `public string $declarationsBlock` when support for <= 7.3 is dropped
* review whether one should use `@readonly` when `vimeo/psalm` is upgraded as part of #793